src/style.css

```diff
@@ -1 +1,90 @@
 /* Styles go here */
+html {
+    --active-color: #00acdc;
+    --font-color: black;
+    --bg-color: #fff;
+}
+
+body {
+    margin: 0;
+}
+
+header {
+    display: flex;
+    padding: 0 50px;
+    font-family: Roboto,sans-serif;
```

Comment:
set it for html

Reply:
I've set just font family for html, because the other 2 break the layout